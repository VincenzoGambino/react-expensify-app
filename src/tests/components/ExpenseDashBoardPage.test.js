import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashbboardPage from '../../components/ExpenseDashboardPage';

test('should render EspenseDashBoardPage component', () => {
  const wrapper = shallow(<ExpenseDashbboardPage />);
  expect(wrapper).toMatchSnapshot();
});