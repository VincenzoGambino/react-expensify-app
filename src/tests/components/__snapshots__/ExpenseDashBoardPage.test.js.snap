// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render EspenseDashBoardPage component 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseSummary) />
  <Connect(ExpenseList) />
</div>
`;
